### PR TITLE
docs: refine contributing guidance and link to readme

### PR DESCRIPTION
**Pull request by barriebyron**
To make our doc update process available to the wider Mina community, this PR iterates on improvements to our contributing guidelines.

This PR moves the build details and CI workflow to https://github.com/o1-labs/docs2/blob/main/README.md and is a 
companion PR to https://github.com/o1-labs/docs2/pull/363

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,12 +20,14 @@ To learn about ways to participate and interact with community members, see [Onl
...
 All contributors agree to respect and follow the Mina Protocol [Code of Conduct](https://github.com/MinaProtocol/mina/blob/develop/CODE_OF_CONDUCT.md).
 
-## Docs Toolset 
+## Docs Toolset
 
 The [Mina Docs](https://docs.minaprotocol.com/) website is published with the Docusaurus static site generator. The docs source files are written in Markdown `.mdx` files that support Markdown with embedded components through the JSX syntax extension to JavaScript. Docusaurus has built-in support for MDX that renders the JSX in Markdown files as React components. 
 
 Even if you're unfamiliar with Markdown, you can still contribute to the docs in a helpful and meaningful way. Use [Markdown Basic Syntax](https://www.markdownguide.org/basic-syntax) and look at existing docs for models to follow. You can start contributing by submitting fixes for typos, spelling, and broken links. 
 
+You can even build the docs locally. See the [README](readme) file.
```
Comment (nicc) [inline CONTRIBUTING.md:29]: Hmm, I wonder about this one. I appreciate and agree with the sentiment but that 'even' is a bit jarring to me. It implies that this is novel, whereas I think it's reasonable to expect that docs can be built locally. I'd suggest something simpler and more direct like "To build the docs locally during development, see the README file.". What do you think?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,25 +47,23 @@ Some of the best content contributions come during the PR review cycles. To ensu
 
 ## Pull Requests (PRs)
 
-Provide a helpful, informative PR title that include the context and page you are updating. Each PR must address one issue or feature.
+To provide a fix or update, make a direct contribution. We appreciate your contributions!
```
Comment (nicc) [inline CONTRIBUTING.md:50]: Stylisitic nitpick: the repitition of 'contribution[s]' breaks flow a little.
Here's a suggestion but of course feel free to improve!
> Feel free to submit a fix or update directly. We appreciate your contributions!

**Review by nicc — APPROVED** [review]
Added a couple of small editorial suggestions but nothing serious.

File: CONTRIBUTING.md (hunk 2 above)
Comment (barriebyron) [reply CONTRIBUTING.md:50]: oops yes of course

**Pull request merged**